core/templates/dev/head/pages/base.html

```diff
@@ -31,7 +31,7 @@
 {%- endmacro %}
 
 <!DOCTYPE html>
-<html ng-app="oppia" ng-controller="Base" itemscope itemtype="http://schema.org/Organization">
+<html ng-app="oppia" update-language lang="en" ng-controller="Base" itemscope itemtype="http://schema.org/Organization">
```

Comment:
Just a question: does setting the default language to 'en' here mean that, if I log in, change my preferred language to something else, and then clear the cache and reload the page, the language here would still be 'en'? Worth checking this, I think -- otherwise, perhaps there's some kind of 'translate init' event that also needs to be listened to in the directive.